pywbem/_logging.py

```diff
@@ -32,53 +32,103 @@
 
 * ``pywbem.ops`` logs user-issued calls to pywbem WBEMConnection methods that
   drive WBEM operations (see :ref:`WBEM operations`). Log
-  entries are created at INFO log level. This logs the parameters of each
+  entries are created at DEBUG log level. This logs the parameters of each
   request and the CIMObjects/exceptions in each method repoonse.
 
 * ``pywbem.http`` logs HTTP requests and responses between the pywbem client
-  and WBEM servers, at the info level. This logs the http request data and
+  and WBEM servers, at the DEBUG level. This logs the http request data and
   response data. This named logger is also defined to create log entries when
   logging is at the INFO log level.
```

Comment:
The last sentence "This named logger is also defined to create log entries when logging is at the INFO log level." is now in conflict with the change to DEBUG level.

Reply:
Fixed